pkgs/by-name/ca/castero/package.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, fetchFromGitHub
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "castero";
+  version = "0.9.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xgi";
+    repo = pname;
```

Comment:
usage of pname is considered anti-pattern

https://github.com/NixOS/nixpkgs/issues/277994